iphone/TitaniumKit/TitaniumKit/Sources/Kroll/KrollContext.m

```diff
@@ -148,7 +148,13 @@ static JSValueRef CommonJSRequireCallback(JSContextRef jsContext, JSObjectRef js
     return [KrollObject toValue:ctx value:result];
   }
   @catch (NSException *e) {
-    return ThrowException(jsContext, [e reason], exception);
```

Comment:
I think the `ThrowException` method should be modified to report the exception. So all exceptions are logged.

```Obj-C
JSValueRef ThrowException(JSContextRef contextRef, NSString *message, JSValueRef *exception)
{
  JSGlobalContextRef globalContextRef = JSContextGetGlobalContext(contextRef);
  JSContext *context = [JSContext contextWithJSGlobalContextRef:globalContextRef];
  JSValue *errorValue = [JSValue valueWithNewErrorFromMessage:message inContext:context];
  [[TiExceptionHandler defaultExceptionHandler] reportScriptError:[TiUtils scriptErrorValue: @{
    @"stack": [errorValue valueForProperty:@"stack"],
    @"message": [errorValue valueForProperty:@"message"]
  }]];

  JSStringRef jsString = JSStringCreateWithCFString((CFStringRef)message);
  *exception = JSValueMakeString(contextRef, jsString);
  JSStringRelease(jsString);
  return JSValueMakeUndefined(contextRef);
}
```

Reply:
Actually, maybe you could just do this:
```Obj-C
JSValueRef ThrowException(JSContextRef contextRef, NSString *message, JSValueRef *exception)
{
  JSGlobalContextRef globalContextRef = JSContextGetGlobalContext(contextRef);
  JSContext *context = [JSContext contextWithJSGlobalContextRef:globalContextRef];
  *exception = [JSValue valueWithNewErrorFromMessage:message inContext:context];
  return JSValueMakeUndefined(contextRef);
}
```
I haven't tested this, just an idea.

Reply:
Yes. Good idea. Updated PR.